src/components/forms/formComponents.js

```diff
@@ -175,59 +177,46 @@ export const formInputMultiText = (config) => {
   ]);
 };
 
+// Using react-select/creatable - Passing config directly through!
 export const formInputSelect = (config) => {
   const {
-    id, title, disabled, error, setError,
+    id, title, disabled, required, error, setError,
     selectOptions, placeholder, ariaDescribedby,
-    creatableConfig
+    formValue, creatable, isMulti, setFormValue,
```

Comment:
I understand why changing the creatableConfig variable for the `isCreatable` variable is simpler, but at the same time, I would prefer to be able to pass in the creatable config as well - There is a portion of the researcherInfo component that uses the creatable and overrides certain parts of the config features in a way that I do not think should be part of the core common forms code. (needs access to the getOptionLabel, and passes in a function that transforms multiple existing fields on the same object).

I think it would be really messy to pull out all of the creatable config options into the top level, and it would be cleaner to just pass through a straight obj, regardless of what fields it contains